en/django_installation/instructions.md

```diff
@@ -185,6 +185,7 @@ First create a `requirements.txt` file inside of the `djangogirls/` folder, usin
 djangogirls
 └───requirements.txt
 ```
+> **Note**: in your directory structure, you will also see your `venv` directory.
```

Comment:
If we stick to the note approach, please start the sentence with a capital letter:

```suggestion
> **Note**: In your directory structure, you will also see your `venv` directory.
```

Reply:
I prefer to modify the directory tree rather than writing notes. I fixed the pull request.